Makefile

```diff
@@ -298,6 +298,28 @@ install-npm-remarklint: install-npm    ## install remark-lint dependency for 'ch
 		 	remark-preset-lint-markdown-style-guide \
 	)
 
+y.PHONY: install-transform
+install-transform: install-cairo-dependencies       # install-trfm-dependencies install-gdal
+
+
+#.PHONY: install-gdal
+#install-gdal:    ## install GDAL
+#	@echo "Installing conda packages for GDAL"
+#	@bash -c '$(CONDA_CMD) conda install -c conda-forge gdal'
+#	@echo "Install GDAL complete."
+#
+#.PHONY: install-trfm-dependencies
+#install-trfm-dependencies:   ## install required dependencies for Transformer
+#	@echo "Install required dependencies for Transformer"
+#	@bash -c '$(CONDA_CMD) pip install --upgrade -r "$(APP_ROOT)/requirements-trfm.txt"'
+#	@echo "Install with pip complete."
+
+.PHONY: install-cairo-dependencies
+install-cairo-dependencies:   ## install required dependencies for Transformer
+	@[ -f "$(shell which cairo)" ] || ( \
+		echo "Binary package manager cairo not found. Attempting to install it."; \
+		apt-get install libpangocairo-1.0-0 \
```

Comment:
Is this absolutely necessary? Installing the `cairosvg` package does not install its dependency for you? 

I expect the Docker image will break otherwise, since this won't be installed.